Gemfile

```diff
@@ -1,3 +1,9 @@
 source 'https://rubygems.org'
 
+group :development, :test do
+  gem 'simplecov', require: false
+  gem 'rubocop', '~> 0.37.2', require: false unless RUBY_VERSION =~ /^1.8/
```

Comment:
I think we could use codeclimate to do the check instead of running it ourselves. Formtastic is already there. It is just matter of enabling rubocop check.